mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMessageWithAttachmentMapperTest.java

```diff
@@ -37,6 +38,7 @@
 
     @BeforeClass
     public static void setUpClass() {
+        DockerCassandraSingleton.restart();
```

Comment:
This should be handled by the rule, no?

Reply:
Not really : 
 - Restarting too much is not good for build time. Doing it on every class would hurt (singleton strategy)